JSDemos/Demos/Charts/PolarChartAnnotations/description.md

```diff
@@ -1,12 +1,12 @@
 Annotations are images and text blocks that provide additional information about the visualized data.
```

Comment:
Let's replace "provide" with "display". Ray mentioned several times that we overuse "provide" and not always use it appropriately. 

```suggestion
Annotations are images and text blocks that display additional information about the visualized data.
```